public/app/plugins/panel/table/README.md

```diff
@@ -6,4 +6,4 @@ The table panel is very flexible, supporting both multiple modes for time series
 
 Check out the [Table Panel Showcase in the Grafana Playground](http://play.grafana.org/dashboard/db/table-panel-showcase) or read more about it here:
 
-[https://grafana.com/docs/grafana/latest/features/panels/table_panel/](https://grafana.com/docs/grafana/latest/features/panels/table_panel/)
+[http://docs.grafana.org/reference/table_panel/](http://docs.grafana.org/reference/table_panel/)
```

Comment:
This seems to happen automatically.... for this README I think the modification is wrong